src/porepy/numerics/fv/fv_elliptic.py

```diff
@@ -142,6 +142,10 @@ def assemble_matrix(self, g, data):
 
         div = pp.fvutils.scalar_divergence(g)
         flux = matrix_dictionary["flux"]
+        if flux.shape[0] != g.num_faces:
```

Comment:
Is this a good way of doing this? I would prefer having a keyword that specifies that the operation should be done, if possible? 
Anyhow, method documentation  is necessary, so is comments in the code for anyone not very familiar with the mpfa implementation.

Reply:
I think this is the most consistent way of defining it as of now. Specifying values on subfaces is for expert users anyhow. I think you are right about including it as a keyword, but I think it is necessary to have a new discussion about the parameter class an whether this fits into that one.

Reply:
Then, can you leave it as a implementation note at the top of some function (not in the method documentation, but right below)?